CHANGELOG.adoc

```diff
@@ -30,6 +31,95 @@ Caused by:
 `quill generate` currently outputs .pem files without an `EC PARAMETERS` section.
 `dfx identity import` will now correctly identify these as EC keys, rather than Ed25519.
 
+=== feat: Remote canister support
+
+It's now possible to specify that a canister in dfx.json references a "remote" canister on a specific network,
+that is, a canister that already exists on that network and is managed by some other project.
+
+Motoko, Rust, and custom canisters may be configured in this way.
+
+This is the general format of the configuration in dfx.json:
```

Comment:
Is this a complete minimal configuration? I.e., I don't need to set any additional properties to get something dfx likes?

Reply:
No, this is only the complete `remote` field, not a complete `dfx.json`.  A Motoko canister would need a `src` field, for example. And a `defaults`, `networks`, and `version` would also be required.

Reply:
Sorry, I wasn't very clear - I meant if this is a minimal complete `canister` section?

Reply:
No, this shows the form of the `remote` section.  A complete `canister` section would be different for a Motoko, Rust, or custom canister.  Below are examples of a complete custom or Motoko section.